go.mod

```diff
@@ -143,6 +143,6 @@ require (
 // github.com/operator-framework/operator-sdk.
 replace (
 	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20120707110453-a547fc61f48d
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20240522145529-93d6bda14341
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20240704111439-6eb7973a3a45
```

Comment:
Personally, I see the benefit of adding the vendored files into a separate commit only if this facilitates the review process (a lot of updates in `vendor/`). For the building process, most of the binaries we build use `-mod=vendor` therefore expecting the vendor directory to contain all the changes needed. Putting the vendor commit first may result into a broken commit too (e.g. [api crd package move](https://github.com/openshift/cluster-ingress-operator/commit/65dcb3f49c7c85f6ac98dd65e679beec2a3198e1)).

Reply:
However for the backporting, a separate vendor commit can indeed be helpful. It can reduce the burden of cherry picking dependencies from a different branch. So, for this PR, I agree it makes sense. Let me extract it.